inlong-agent/conf/agent.properties

```diff
@@ -14,9 +14,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
```

Comment:
It is recommended not to format the properties file. Appropriate spaces can increase the readability of the file.

Reply:
resolve